stream/statelib/src/main/java/org/apache/bookkeeper/statelib/impl/kv/RocksdbKVStore.java

```diff
@@ -275,7 +275,7 @@ protected void openRocksdb(StateStoreSpec spec) throws StateStoreException {
         // initialize the write options
 
         writeOpts = new WriteOptions();
-        writeOpts.setDisableWAL(false); // disable wal, since the source of truth will be on distributedlog
+        writeOpts.setDisableWAL(true); // disable wal, since the source of truth will be on distributedlog
```

Comment:
Add a comment in order to explain this.
I guess I know the reason but better to be clear

Reply:
👍, seems not related but right fix.

Reply:
The comment is already there, no? It was set to a wrong flag.

Reply:
@eolivelli I replied to your comment